1_fetch/src/fetch_river_geometry.R

```diff
@@ -1,16 +1,17 @@
-fetch_river_geoms <- function(ind_file, view_polygon, sites, streamorder) {
+fetch_river_geoms <- function(ind_file, view_polygon, sites_ind) {
 
-  sites <- readRDS(sc_retrieve(sites))
+  sites <- readRDS(sc_retrieve(sites_ind))
 
   if(nrow(sites) > 30) {
     set.seed(42)
     sites <- sites[sample(seq_len(nrow(sites)), 30),]
   }
```

Comment:
This is just a temporary fix so we don't select so many rivers?

Reply:
Correct. I hit the NLDI for each site and having a long list becomes problematic.